contracts/bprotocol/avatar/AbsCToken.sol

```diff
@@ -211,6 +211,14 @@ contract AbsCToken is Cushion {
         return cToken.approve(spenderAvatar, amount);
     }
 
+    function collectCToken(ICToken cToken, address from, uint256 cTokenAmt) public postPoolOp(false) {
+        // `from` should not be an avatar
+        require(registry.ownerOf(from) == address(0), "AbsCToken: from-is-an-avatar");
+        require(cToken.transferFrom(from, address(this), cTokenAmt), "AbsCToken: transferFrom-failed");
+        uint256 underlyingAmt = _toUnderlying(cToken, cTokenAmt);
+        _score().updateCollScore(address(this), address(cToken), toInt256(underlyingAmt));
```

Comment:
need to handle quit

Reply:
fixed in https://github.com/backstop-protocol/BCompound/pull/49/commits/747892efcfd53f060bdc90edab4bd130822a47df